src/events/messageCreate.ts

```diff
@@ -0,0 +1,126 @@
+import { EmbedBuilder, type TextChannel, type Message } from "discord.js";
+import { pathToFileURL } from "url";
+import { join } from "path";
+import { readdirSync } from "fs";
+import { genColor } from "../utils/colorGen.js";
+import { database, getLevelingTable, getSettingsTable } from "../utils/database.js";
+import { Reward } from "../commands/settings/leveling/rewards.js";
+
+export default {
+  name: "messageCreate",
+  event: class MessageCreate {
+    db = null;
+
+    async run(message: Message) {
+      // Easter egg handler
+      if (message.author.bot) return;
+      if (message.guildId !== "903852579837059113") return;
+      const eventsPath = join(process.cwd(), "src", "events", "easterEggs");
+
+      for (const easterEggFile of readdirSync(eventsPath)) {
+        const msg = await import(pathToFileURL(join(eventsPath, easterEggFile)).toString());
+        new msg.default().run(message, ...message.content);
+      }
```

Comment:
Yo, another comment 👀  but I like that you put it in here anyways. Comment here is fine 👍

Reply:
Same for levelling of course

Reply:
yeah it was 2 separate files so now that its in one i have to comment what is what to avoid confusion